src/System.Data.Odbc/tests/Helpers.cs

```diff
@@ -18,9 +18,11 @@ public static class Helpers
 
         public static bool CheckOdbcNotAvailable() => !CheckOdbcIsAvailable();
 
-        private static bool CheckOdbcIsAvailable() => 
-            PlatformDetection.IsWindows ?
-                !PlatformDetection.IsWindowsNanoServer && (!PlatformDetection.IsWindowsServerCore || Environment.Is64BitProcess) :
+        private static bool CheckOdbcIsAvailable() =>
+#if TARGETSWINDOWS
```

Comment:
What was wrong with this before? It would be useful to submit the ODBC fix separately since it has not much to do with the rest.

Reply:
The issue was that [this line](https://github.com/dotnet/corefx/pull/26229/files/472ee13954bb706775be731e2295b7e51b1e0296#diff-a143ee08bff6738f1b57f467f72e4f58R25) was being included in the windows test build which meant we needed to also include the linux interop files in the windows build. That doesn't really make a ton of sense (and it led to some conflicts) so I made it so the unix interop call is only included in the unix build.

Reply:
ok